doc/source/user_manual/configuration/processor.rst

```diff
@@ -2,6 +2,50 @@
 Processors
 ==========
 
+
+Common Configurations
+=====================
+
+Most processors share the following configurations.
+The only exceptions at this point are the
+processors :code:`Domain Resolver`, :code:`Dropper` and :code:`Selective Extractor`.
+These three processors do not yet support generic and specific rules.
+
+type
+----
+
+The type value defines the processor type that is being configured.
+The exact values that are currently possible are given at the specific processor
+configurations below.
+
+generic_rules
+-------------
+
+List of directory paths with generic rule files that can match multiple event types.
+These rules are being executed before specific rules, i.e.:
```

Comment:
I changed this for the time being, as this order is still subject to discussion and might change.